artifactory.py

```diff
@@ -1015,14 +1037,27 @@ def del_properties(self, pathobj, props, recursive):
     def scandir(self, pathobj):
         return _ScandirIter((pathobj.joinpath(x) for x in self.listdir(pathobj)))
 
-    def writeto(self, fd, out, chunk_size):
-        url = str(fd)
-        res = fd.session.get(url, stream=True, verify=True, cert=None)
-        if res.status_code != 200:
-            raise RuntimeError(res.status_code)
-        for chunk in res.iter_content(chunk_size=chunk_size):
-            if chunk:
-                out.write(chunk)
+    def writeto(
+        self, pathobj, file, chunk_size=1024, progress_func=print_download_progress
```

Comment:
We define `progress_func` default value twice. Do we need it here?

Reply:
The same question about `chunk_size`

Reply:
I was also thinking about this, only case I see, when users will develop their own class and logic and this method will be inherited

However then they have to have the same function also declared before the class

Do you suggest to remove?

Reply:
I removed it